src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -179,29 +230,30 @@ export const SpendReport = ({ tab, billingProject }) => {
           div({ style: { gridRowStart: 1, gridColumnStart: 1 } }, [
             h(IdContainer, [id => h(Fragment, [
               h(FormLabel, { htmlFor: id }, ['Date range']),
-              h(Select, {
+              h(NumberSelect, {
                 id,
                 value: spendReportLengthInDays,
                 options: _.map(days => ({
                   label: `Last ${days} days`,
                   value: days
                 }), [7, 30, 90]),
-                // @ts-ignore
-                onChange: ({ value: selectedDays }) => {
+                onChange: selectedOption => {
+                  const selectedDays = selectedOption!.value
```

Comment:
I'm not sure why the not-null assertion is necessary here.
When I have intellij display the type of selectedOption, it doesn't show up as nullable, but when I remove it, I get a compile error...
So while it isn't ideal, I think you're correct in just putting the not null assertion and moving on, for now.

Reply:
I'm not sure either, but @nawatts sent me the code, so I happily used it. :)

Reply:
This type comes from the underlying react-select library.

https://github.com/JedWatson/react-select/blob/597143ee3d5c27ce87e9b1508b068dc7c1323201/packages/react-select/src/types.ts#L15-L21

I'm not sure what causes onChange to be called with null, or if this is an artifact of onChange using the same type as the value prop (which accepts null for "no selection").